### PR TITLE
Add a configuration option to reuse Window

### PR DESCRIPTION
**Pull request by tkf**
This requires https://github.com/davidanthoff/Electron.jl/pull/28

fixes https://github.com/fredo-dedup/VegaLite.jl/issues/141

### CODE REVIEW

**Comment by davidanthoff** [comment]
Perfect, I'll review this once we have the Electron.jl thing merged!

File: src/ElectronDisplay.jl [hunk 1]
```diff
@@ -4,21 +4,57 @@ using Electron, Base64
...
+"""
+const CONFIG = ElectronDisplayConfig(false)
+
+const _window = Ref{Window}()
+
+function _getglobalwindow()
+    if !(isdefined(_window, 1) && _window[].exists)
+        _window[] = Electron.Window(
+            URI("about:blank"),
+            options=Dict("webPreferences" => Dict("webSecurity" => false)))
```
[the remark this answers is not in the document]
Comment (tkf) [reply src/ElectronDisplay.jl]: Yes, as VegaLite output is not rendered without passing this option.

[the remark this answers is not in the document]
Comment (tkf) [reply src/ElectronDisplay.jl]: (By "Yes", I mean `"webSecurity" => false` simplifies things for VegaLite)

[the remark this answers is not in the document]
Comment (davidanthoff) [reply src/ElectronDisplay.jl]: But the current code works, right? So this shouldn't be a reason not to merge?

[the remark this answers is not in the document]
Comment (tkf) [reply src/ElectronDisplay.jl]: Yes, current implementation works.  I checked it with VegaLite.jl and also `display("image/png", plot(sin.(0:0.1:3)))` with Plots.jl

**Pull request merged**